lib/chef-vault/chef_api.rb

```diff
@@ -0,0 +1,39 @@
+# Author:: Tyler Cloke <tyler@chef.io>
+# Copyright:: Copyright 2016, Chef Software, Inc.
+# License:: Apache License, Version 2.0
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "chef/server_api"
```

Comment:
Different API versions where added when I was planning on supporting Chef Server 12.0-12.5 instead of just 12.5+. If we decide it is too difficult to support 12.0-12.5 along side 12.5+, I'll simplify this a lot.